### PR TITLE
Update AUTHORS.txt

### PR DESCRIPTION
**Pull request by jplanasc**

### CODE REVIEW

File: AUTHORS.txt
Comment (sergiorg-hpc) [inline AUTHORS.txt:11]: I would also add `Ankur Sinha` (#182) and others from:
https://github.com/BlueBrain/libsonata/graphs/contributors

Comment (jplanasc) [reply AUTHORS.txt:11]: I agree we should revise who has contributed. I'd probably make a distinction, though between authors and contributors.

Comment (sergiorg-hpc) [reply AUTHORS.txt:11]: Your call! If you need any help, just let me know and I'll be glad, ok?

Comment (jplanasc) [reply AUTHORS.txt:11]: From the contributors graph, only Nadir and Ankur Sinha are missing in this file.
Since @mgeplf is the maintainer and he's the only one who has been updating this file, I leave it to him.

Comment (mgeplf) [reply AUTHORS.txt:12]: Ankur Sinha's contributions have been very helpful, so I'll add them.

Thanks for the updates!

Comment (mgeplf) [reply AUTHORS.txt:12]: (was off, so didn't respond earlier, sorry)

**Review by sergiorg-hpc — APPROVED** [review]

**Review by mgeplf — APPROVED** [review]

**Pull request merged**
